### ISSUE TITLE
Monkey-patching suds-jurko

### ISSUE THREAD

**Issue by andreycizov** [googleads/googleads-python-lib#252]
Unfortunately googleads-python-lib monkey patches suds-jurko when imported, thus disabling the user from using suds-jurko for anything but Google Ads.

Is there any possible case this could be solved without monkey-patching a library that could be used outside your context?

**Comment by andreycizov** [comment]
The regression appeared in 10.0.0 as the code responsible for monkey-patching exists in 9.0.0 but somehow does not cause any interference.

**Comment by msaniscalchi** [comment]
Hello,

I'll follow up to some of the specific points you bring up:

> The regression appeared in 10.0.0 as the code responsible for monkey-patching exists in 9.0.0 but somehow does not cause any interference.

What exactly is the regression appearing in 10.0.0?

>Is there any possible case this could be solved without monkey-patching a library that could be used outside your context?

We've discovered bugs and missing features. We've reported these to suds-jurko and had some limited success in getting the project owner to integrate fixes into the 0.7 release. However, that release remains in development and was never published externally. Since then, I would now consider the library "mostly abandoned" by the owner.

That said, we do plan to move to the zeeps library in the near-ish future. It is much better maintained, so I don't expect we'll have longstanding bugs that we'll be forced to patch ourselves. There will be a period where we support both temporarily.

> thus disabling the user from using suds-jurko for anything but Google Ads.

In what way does the patch prevent users from using suds-jurko outside the context of Google Ads?
